packages/clay-date-picker/src/index.tsx

```diff
@@ -344,36 +344,59 @@ const ClayDatePicker: React.FunctionComponent<IProps> = React.forwardRef<
 		const inputChange = (event: React.ChangeEvent<HTMLInputElement>) => {
 			const {value} = event.target;
 
-			const format = time
-				? `${dateFormat} ${use12Hours ? TIME_FORMAT_12H : TIME_FORMAT}`
-				: dateFormat;
+			if (!value) {
+				const newDaysSelected: [Date, Date] = [NEW_DATE, NEW_DATE];
```

Comment:
For this one you would need to use `initialMonth` as well because it can happen that the developer sets a date different from the current date he would like, this is common for Forms for example.

```suggestion
				const newDaysSelected: [Date, Date] = [initialMonth, initialMonth];
```

Reply:
Ohh understood! I thought otherwise, the developer could change `NEW_DATE` constant